### ISSUE TITLE
Dot notation in key name

### ISSUE THREAD

**Issue by benrolfe** [Tucker-Eric/EloquentFilter#71]
I'm passing through the following search criteria, but it's only searching by my name "Ben", the company name is being ignored. How do I use dot notation?

```
[
    'Name'  => 'Ben',
    'Company.Name'  => 'Nike',
]
```

Here's my PHP code:
```
// works
public function name($value)
{
    return $this->where('Name', 'LIKE', "%$value%");
}

// ignored
public function companyName($value)
{
    return $this->related('company', 'Name', 'LIKE', "%$value%");
}
```

**Comment by Tucker-Eric** [comment]
This currently is not supported but I'd be happy to merge this feature if you submitted a pr for it.

**Comment by benrolfe** [comment]
@Tucker-Eric Do you know what the issue is? Is the key ignored because of the dot? Does it throw a silent error? Does it try to match against an odd/invalid method name (eg: company.Name())?

**Comment by Tucker-Eric** [comment]
Yup, it tries to match an invalid method name `company.Name()`. Looks like Laravel's `camel_case` helper method doesn't strip dots.

I've just updated and published fix in [1.3.1](https://github.com/Tucker-Eric/EloquentFilter/releases/tag/1.3.1). You should be good to go with those updates.
